ea_airflow_util/providers/aws/operators/s3.py

```diff
@@ -93,3 +95,114 @@ def execute(self, context):
             transferred_keys.append(self.dest_s3_key)
 
         return transferred_keys
+    
+
+class S3ToSnowflakeOperator(BaseOperator):
+    """
+    Copy JSON files saved to S3 to Snowflake raw tables.
+
+    The optional custom_metadata_columns param takes a dictionary in the format {alias: value}
+    """
+    template_fields = ('s3_destination_key', 's3_destination_dir', 's3_destination_filename',)
+
+    @apply_defaults
+    def __init__(self,
+        *,
+        snowflake_conn_id: str,
+        database: str,
+        schema: str,
+        table_name: str,
+        custom_metadata_columns: Optional[dict] = None,
+
+        s3_destination_key: Optional[str] = None,
+        s3_destination_dir: Optional[str] = None,
+        s3_destination_filename: Optional[str] = None,
+
+        full_refresh: bool = False,
+        delete_where: Optional[str] = None,
+        xcom_return: Optional[Any] = None,
+        **kwargs
+    ) -> None:
+        super(S3ToSnowflakeOperator, self).__init__(**kwargs)
+
+        self.snowflake_conn_id = snowflake_conn_id
+        self.database = database
+        self.schema = schema
+        self.table_name = table_name
+        self.custom_metadata_columns = custom_metadata_columns
+
+        self.s3_destination_key = s3_destination_key
+        self.s3_destination_dir = s3_destination_dir
+        self.s3_destination_filename = s3_destination_filename
+
+        self.full_refresh = full_refresh
+        self.delete_where = delete_where
```

Comment:
Should this delete where include the "WHERE" keyword, or just have the filter clause?

Reply:
Right now it expects a "WHERE" keyword to be included. Do you have an opinion on whether it should or should not?